app/controllers/mixins/actions/vm_actions/reconfigure.rb

```diff
@@ -210,19 +211,10 @@ def reconfigure_calculations(mbsize)
           return humansize.to_s, fmt
         end
 
-        def get_vlan_options(host_id)
-          vlan_options = []
-
-          # determine available switches for this host...
-          switch_ids = []
-          Rbac.filtered(HostSwitch.where("host_id = ?", host_id)).each do |host_switch|
-            switch_ids << host_switch.switch_id
-          end
-
-          Rbac.filtered(Lan.where("switch_id IN (?)", switch_ids)).each do |lan|
-            vlan_options << lan.name
-          end
-          vlan_options.sort
+        # determine available switches for this host...
```

Comment:
This comment doesn't make sense on the method - the method doesn't return the switches for the host, but instead returns the vlans on those switches